docs/parser.md

```diff
@@ -273,6 +273,28 @@ You should enable these features only if you are using an older version.
     Some code has different meaning in Flow and in vanilla JavaScript. For example, `foo<T>(x)` is parsed as a call expression with a type argument in Flow, but as a comparison (`foo < T > x`) accordingly to the ECMAScript specification. By default, `babel-parser` parses those ambiguous constructs as Flow types only if the file starts with a `// @flow` pragma.
     Set this option to `true` to always parse files as if `// @flow` was specified.
 
+### Error codes
+
+Error codes have been introduced in `7.14.0`. This is useful if you want to handle the errors thrown by babel-parser by type.
```

Comment:
We can use an history table to say when they have been introduced. Also `babel-parser` -> `@babel/parser` (and it should be quoted with `)